addons/mail/models/mail_thread.py

```diff
@@ -1985,7 +1985,8 @@ def _notify_thread(self, message, msg_vals=False, **kwargs):
             message_values['channel_ids'] = [(6, 0, [r['id'] for r in rdata['channels']])]
 
         self._notify_record_by_inbox(message, rdata, msg_vals=msg_vals, **kwargs)
-        self._notify_record_by_email(message, rdata, msg_vals=msg_vals, **kwargs)
+        if not kwargs.get('skip_channel_mail'):
```

Comment:
Why not explicit arg with defaults?
I would prefer a more meaningful name, something like `notify_by_email`. It defaults to `True`, and invite calls `message_post` with this arg set to `False`.

Reply:
@alexkuhn Done